src/pseudopeople/interface.py

```diff
@@ -14,15 +16,28 @@
 from pseudopeople.schema_entities import COLUMNS, DATASETS, Dataset
 from pseudopeople.utilities import configure_logging_to_terminal, get_state_abbreviation
 
+if TYPE_CHECKING:
+    import modin.pandas
+
+    # TODO: Using type aliases isn't very good for autodoc purposes,
+    # since it displays the alias and not its actual value.
+    # When we have more engines and these types are more complicated,
+    # we might consider exposing them as a public module on pseudopeople,
+    # then referencing them.
+    # But for now, we just duplicate the full types everywhere so there
+    # is less indirection in the docs.
+    # from pseudopeople.utilities import DATAFRAME, ENGINE
+
 
 def _generate_dataset(
```

Comment:
This is a "private" method meaning it shouldn't be getting autodocs, so the concern above related to type aliases doesn't really apply here.